apps/backoffice-v2/index.html

```diff
@@ -8,6 +8,7 @@
     <link rel="manifest" href="/manifest.webmanifest" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Ballerine - Backoffice</title>
+    <script type="text/javascript" src="/config.js"></script>
```

Comment:
I'm not sure this is needed since `env.ts` is imported very early on and the config file is imported there.

Reply:
I think this is needed else the config changes are not getting rendered